### ISSUE TITLE
Lack of a comprehensive summary of all commands in UG

### ISSUE THREAD

**Issue by OrangeCatLoves** [OrangeCatLoves/pe#6]
As I was trying to have a quick overview of all the commands and its various formats in your UG, I can't seem to find it anywhere in your UG. I think it's rather important to have a table summarising all the instructions and it's format
It can go something like this:
Name                                                          Command                     Parameters
Create a new tutorial                                createtut                        tut/
Enroll a student in a tutorial                    enroll                              <index> tut/

Basically a summary of all commands consolidated in a table in your UG, so at a glance I can see all your commands in a unified area instead of having to scroll back and forth. This is especially a problem for people who may not have yet been familiarised with all your commands and have to scroll back and forth to search for the structure of your command. It's best to have a summary table so users can just keep referring to the same table without much cumbersome scrolling.
